flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/functions/sql/ProctimeSqlFunction.java

```diff
@@ -37,16 +39,24 @@ public ProctimeSqlFunction() {
 		super(
 			"PROCTIME",
 			SqlKind.OTHER_FUNCTION,
-			ReturnTypes.explicit(new ProctimeRelProtoDataType()),
-			null,
-			OperandTypes.NILADIC,
-			SqlFunctionCategory.TIMEDATE);
+			ReturnTypes.cascade(ReturnTypes.explicit(SqlTypeName.TIMESTAMP), SqlTypeTransforms.TO_NULLABLE),
```

Comment:
`PROCTIME()` is a built-in function used to generate proctime attribute field, whose return type must be proctime time indicator.

I think what you want is `ProctimeMaterializeSqlFunction` which materialize a proctime field into `Timestamp` field.

So I think we don't need to change this file. We should use `FlinkSqlOperatorTable.PROCTIME_MATERIALIZE` in `RelTimeIndicatorConverter`.

Reply:
I didn't notice `ProctimeMaterializeSqlFunction`'s existence before, i will revert the changes in `ProctimeSqlFunction`